sdk/src/main/java/co/omise/android/models/Capability.kt

```diff
@@ -36,4 +37,24 @@ data class Capability(
 
         override fun type(): Class<Capability> = Capability::class.java
     }
+
+    companion object {
+
+        /**
+         * The helper function for creating an instance of [Capability].
+         * @param allowCreditCard allow to create a [Token] with a credit card or not. Default is true.
+         * @param sourceTypes list of [SourceType] that allow to create a [Source].
+         * @return an instance of [Capability] with specific configuration.
```

Comment:
```suggestion
         * The helper function for creating an instance of the [Capability] class.
         * @param allowCreditCard allows users to create a [Token] with or without a credit card. Default is true.
         * @param sourceTypes list of [SourceType] that allow users to create a [Source].
         * @return an instance of [Capability] with a specific configuration.
```

Reply:
Some small changes that I think would help improve the docs.

Reply:
OK, I'll make changes.

Reply:
OK, I'll make changes.